src/AddIns/Uno.UI.MediaPlayer.WebAssembly/MediaPlayerExtension.cs

```diff
@@ -292,9 +292,16 @@ public void InitializeSource()
 
 			switch (_owner.Source)
 			{
-				case MediaPlaybackList playlist when playlist.Items.Count > 0 && _playlistItems is not null:
+				case MediaPlaybackList playlist when playlist.Items.Count > 0:
 					SetPlaylistItems(playlist);
-					_uri = _playlistItems[0];
+					if (_playlistItems is not null)
+					{
+						_uri = _playlistItems[0];
```

Comment:
Could it be possible for `_playlistItems` to be **not null** but **an empty array**, causing an index-out-of-range exception?

Reply:
as it is a list of Uri.
`private List<Uri>? _playlistItems;`
we can get a list of default Uri. 
But not a Exception

Reply:
But i add a check for Any(). Just in case.